File: PRAK603-2310817310008-MUHAMMADRIZKIRAMADHAN.py
pertama, kedua = map(int, input().split())
baris1 = []
baris2 = []
if (pertama != kedua):
    print("Jumlah tidak sama")
else:
    barispertama = input().split()
    bariskedua = input().split()
    for i in range(0,pertama):
        baris1.append(int(barispertama[i]))
        baris2.append(int(bariskedua[i]))
        for i in range(pertama):
            print(baris1[i] * baris2[i], end=" ")

File: PRAK603-2310817310008-MUHAMMADRIZKIRAMADHAN.c
#include <stdio.h>

int main(){
    int pertama, kedua;
    printf(" ");
    scanf("%d %d", &pertama, &kedua);
    int baris1[pertama], baris2[kedua];
    if (pertama != kedua){
        printf("Jumlah tidak sama\n");
    }
    else{
        for (int i = 0; i < pertama; i++){
        scanf("%d", &baris1[i]);
        }
        for (int i = 0; i < kedua; i++){
        scanf("%d", &baris2[i]);
        }
        for (int i = 0; i < pertama; i++){
        printf("%d ", baris1[i] * baris2[i]);
        }
    }
    return 0;
}